README.md

```diff
@@ -1 +1,64 @@
-# Module1
\ No newline at end of file
+![](https://github.com/LionRouge1/Module1.git)
+
+# Portfolio
+
+> One paragraph statement about the project.
+
+![screenshot](images/Module.png)
+
+Setup and mobile version skeloton
+
+## Built With
+
+- Html and css
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] [OPTIONAL] Please make sure to remove unused sections in your `README.md` file. Always remember that this file is the first impression that any persona will have about your project and work, this includes hiring managers or any other stuff from any company, so try to keep it as professional as possible!